### PR TITLE
Heading levels

### PR DESCRIPTION
**Pull request by BNewing**
I noticed that the heading levels weren't hierarchical across the site, so this PR goes through all of the pages and updates them. In this PR each page has a h1 and that all following header tags are at the appropriate level.

I did this because screen reader users often use heading levels to navigate the page: https://www.w3.org/WAI/tutorials/page-structure/headings/

In some cases, the heading levels had been used because of the styling that materialize gave them, so I've created a few classes in the CSS file so that the site still looks the same nearly everywhere. I tried to follow other CSS conventions in the project. 

There is one instance where it doesn't match the live site, and screenshots are below. I felt like with the image spacing them out, it looks okay, but happy to change it to the current design on the site if you'd prefer that! (I'm not sure why the icons are appearing on my local site and not the live site)

Contact info for different chapters appears as follows in my PR: screen shot 2018-10-07 at 21 17 00

Icons appearing on top of page for me locally: oxford-strapline

I've gone through and compared it page by page for every page twice on two separate occasions, as I wasn't sure how to test it. In some places, it isn't pixel perfect with margins, but the font-sizes have been kept consistent with the live site :)

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -332,18 +332,18 @@ <h5 class="header time">
...
                 <path d="M5.4 0C2.4 0 0 2.4 0 5.4s2.4 5.4 5.4 5.4 5.4-2.4 5.4-5.4S8.4 0 5.4 0zm3 6.8H4.7V1.7h.7L6 5.4l2.4.6v.8z"></path>
               </svg>
             </span>
             10:00 - 16:00
-          </h5>
+          </p>
           <div class="event">
 
-            <div class="title">
+            <div class="titlep
```
Comment (minaorangina) [inline events.html]: typo: "titlep --> "title"

File: events.html [hunk 2]
```diff
@@ -559,18 +559,18 @@ <h5 class="header time">
...
                 <path d="M5.4 0C2.4 0 0 2.4 0 5.4s2.4 5.4 5.4 5.4 5.4-2.4 5.4-5.4S8.4 0 5.4 0zm3 6.8H4.7V1.7h.7L6 5.4l2.4.6v.8z"></path>
               </svg>
             </span>
             10:00 - 15:00
-          </h5>
+          </p>
           <div class="event">
 
-            <div class="title">
+            <div classptitle">
```
Comment (minaorangina) [inline events.html]: typo: classptitle" --> class="title"

File: events.html [hunk 3]
```diff
@@ -230,20 +230,20 @@ <h5 class="header time">
...
             </span>
             10:00 - 16:00
-          </h5>
+          </p>
           <div class="event">
 
             <div class="title">
               <a target="_blank" class="grey-text" href="https://nodegirls.typeform.com/to/FC57Oj">
-                Introduction to Node.js and JavaScript
+                Introduction topode.js and JavaScript
```
Comment (minaorangina) [inline events.html]: typo: topode --> to Node

File: events.html [hunk 4]
```diff
@@ -281,19 +281,19 @@ <h5 class="header time">
...
               </svg>
             </span>
             10:00 - 16:00
-          </h5>
+          </p>
           <div class="event">
 
             <div class="title">
-              <a target="_blank" class="grey-text" href="https://nodegirls.typeform.com/to/wkVUzc">
+              <a target="_bpnk" class="grey-text" href="https://nodegirls.typeform.com/to/wkVUzc">
```
Comment (minaorangina) [inline events.html]: _bpnk --> _blank

**Review by minaorangina — CHANGES_REQUESTED** [review]
Thanks so much for the PR @BNewing!  I've spotted a couple of small issues and left some comments. Once they're sorted it can be merged 🎉

**Comment by BNewing** [comment]
Sorry about that! Have committed fixing those :)

**Review by minaorangina — APPROVED** [review]

**Pull request merged**